baseplate/context/memcache/lib.py

```diff
@@ -114,15 +102,12 @@ def dump_and_compress(key, value):  # pylint: disable=unused-argument
         :raises ValueError: if `value` is not JSON serializable
 
         """
-        if isinstance(value, string_types):
+        if isinstance(value, str):
```

Comment:
does this mean that bytestrings will get serialized as JSON?

Reply:
this shouldn't be any change in behavior since `string_types` was defined as `(str,)` on py3 already https://github.com/reddit/baseplate.py/pull/283/files#diff-b1da7a5976a089e9c05aae386c2586e3L19

Reply:
ahh ok cool